### PR TITLE
bcachefs: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
I only just started using this filesystem so I'll just list the most basic examples and add more once I see the need for them.

File: pages/linux/bcachefs.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# bcachefs
+
+> Manage bcachefs filesystems/devices.
```
Comment (kbdharun) [inline pages/linux/bcachefs.md:3]:
```suggestion
> Manage `bcachefs` filesystems/devices.
```

File: pages/linux/bcachefs.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://bcachefs.org/bcachefs-principles-of-operation.pdf>.
+
+- Format a partition with bcachefs:
```
Comment (kbdharun) [inline pages/linux/bcachefs.md:6]:
```suggestion
- Format a partition with `bcachefs`:
```

File: pages/linux/bcachefs.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`sudo bcachefs format {{path/to/partition}}`
+
+- Mount a bcachefs filesystem:
```
Comment (kbdharun) [inline pages/linux/bcachefs.md:10]:
```suggestion
- Mount a `bcachefs` filesystem:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Comment by Managor** [comment]
The `--human-readable` option does not currently work as although the help message says it should. It'll work eventually so I'll put that in the page.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages/linux/bcachefs.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Format a partition with `bcachefs`:
+
+`sudo bcachefs format {{path/to/partition}}`
+
+- Mount a `bcachefs` filesystem:
+
+`sudo bcachefs mount {{path/to/partition}} {{path/to/mountpoint}}`
+
+- Create a RAID 0 filesystem where an SSD acts as a cache and an HDD acts as long term storage:
```
Comment (kbdharun) [inline pages/linux/bcachefs.md:14]:
```suggestion
- Create a RAID 0 filesystem where an SSD acts as a cache and an HDD acts as a long term storage:
```

**Pull request merged**